### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by J2ZROMERO**
- Add styles from the Zeplin template.
- Add missing JSX elements (without functionality).

### CODE REVIEW

File: src/styles/Book.css [hunk 1]
```diff
@@ -17,7 +17,7 @@
   width: 50%;
   display: flex;
   flex-direction: column;
-  justify-content: space-evenly;
+  justify-content: space-around;
 }
 
 .percentage {
```
Comment (zieeco) [inline src/styles/Book.css:23]: - Hey, please, let's use the correct color for the image in this div to make the design close to the template as much as we can below:

*original design*
Screenshot from 2022-11-19 05-21-07

*your design*
Screenshot from 2022-11-19 05-20-24


- You can use the react `<CircularProgress
          size={50}
          thickness={2}
          variant="determinate"
        />`
then install and add the `import { CircularProgress } from '@material-ui/core';`

File: src/components/Form.js [hunk 2]
```diff
@@ -32,8 +32,16 @@ const Form = () => {
...
+            <option value="Action">Action</option>
+            <option value="Suspense">Suspense</option>
+            <option value="Romance">Romance</option>
+            <option value="Comedy">Comedy</option>
+            <option value="Thriller">Thriller</option>
+            <option value="Fiction">Fiction</option>
+            <option value="Real Life">Real Life</option>
+          </select>
   <button className="addBook" onClick={addBookButton}>ADD BOOK</button>
         </form>
```
Comment (zieeco) [inline src/components/Form.js:47]: - Hey, Please let's also add the missing header text in the form to match the design as much ass we can below


*original design*
Screenshot from 2022-11-19 05-19-05

*your design*
Screenshot from 2022-11-19 05-18-36

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @J2ZROMERO ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by J2ZROMERO** [comment]
thanks for your review @zieeco

**Comment by J2ZROMERO** [comment]
- Add Circular progress 
- Add  form title (ADD NEW BOOK)

**Review by emmiiorji — APPROVED** [review]
Hi @J2ZROMERO, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats-13

## Good Highlights✔

- Linter checks are passing
- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**